src/Featurizers/CountVectorizerFeaturizer.h

```diff
@@ -26,15 +27,14 @@ class CountVectorizerTransformer : public StandardTransformer<std::string, Spars
     // |
     // ----------------------------------------------------------------------
     using BaseType                           = StandardTransformer<std::string, SparseVectorEncoding<std::uint32_t>>;
-    using IndexMapType                       = std::unordered_map<std::string, std::uint32_t>;
-    using AnalyzerMethod                     = Components::AnalyzerMethod;
-
+    using TfidfEstimator                     = Microsoft::Featurizer::Featurizers::TfidfVectorizerEstimator<std::numeric_limits<size_t>::max()>;
+    using TransformerUniquePtrType           = TfidfEstimator::TransformerUniquePtr;
     // ----------------------------------------------------------------------
     // |
     // |  Public Methods
     // |
     // ----------------------------------------------------------------------
-    CountVectorizerTransformer(IndexMapType map, bool binary, bool lower, AnalyzerMethod analyzer, std::string regex, std::uint32_t ngram_min, std::uint32_t ngram_max);
+    CountVectorizerTransformer(TransformerUniquePtrType pTransformer);
```

Comment:
>pTransformer [](start = 56, length = 12)

Can you update this variable name to make it more clear that this is a transformer created by the TfidfEstimator.